classes/controller/AdminController.php

```diff
@@ -786,7 +786,8 @@ public function processFilter()
         }
 
         $filters = $this->context->cookie->getFamily($prefix.$this->list_id.'Filter_');
-        $definition = ObjectModel::getDefinition($this->className);
+        if (isset($this->className) && $this->className) 
```

Comment:
In case of false statement, $definition may be undefined on line 835 when tested.

I think is good to add this code:
else
    $definition = false;